cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -46,10 +46,27 @@
             </div>
         </div>
     {% endif %}
-    {% if show_topic_filter or value.authors or value.date_range %}
+    {% if show_topic_filter or value.authors or value.statuses or value.date_range %}
         <div class="content-l_col
                     content-l_col-2-3">
             <div class="content-l">
+                {% if value.statuses %}
+                    <div class="content-l_col
+                                content-l_col-1-2">
+                        <div class="o-form_group">
+                            <fieldset class="o-form_fieldset">
+                                <legend class="a-legend">
+                                   Status
+                                </legend>
+                                <ul class="m-list m-list__unstyled">
+                                {% for slug, name in form.statuses.field.choices %}
```

Comment:
If I erroneously check the "Filter Enforcement Statuses" checkbox on a different filterable list page (e.g. the Blog page), this line errors because `form.statuses` doesn't exist. Do you think we should put a guard on this section, even though that situation shouldn't happen?

Reply:
Fixed in 5cb5d9